api.js

```diff
@@ -1,33 +1,11 @@
 'use strict';
-var arrayFindIndex = require('array-find-index');
+var util = require('util');
+var currentlyUnhandled = require('currently-unhandled');
 
 // WARNING: This undocumented API is subject to change.
 
-module.exports = function (process) {
-	var unhandledRejections = [];
-
-	process.on('unhandledRejection', function (reason, p) {
-		unhandledRejections.push({reason: reason, promise: p});
-	});
-
-	process.on('rejectionHandled', function (p) {
-		var index = arrayFindIndex(unhandledRejections, function (x) {
-			return x.promise === p;
-		});
-
-		unhandledRejections.splice(index, 1);
-	});
-
-	function currentlyUnhandled() {
-		return unhandledRejections.map(function (entry) {
-			return {
-				reason: entry.reason,
-				promise: entry.promise
-			};
-		});
-	}
-
+module.exports = util.deprecate(function (process) {
 	return {
-		currentlyUnhandled: currentlyUnhandled
+		currentlyUnhandled: currentlyUnhandled(process)
```

Comment:
It's passing through the parameter (not necessarily `process`, could be a mock for testing). This keeps the old behavior until we just remove it.